Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Image to use 
+FROM python:3
```

Comment:
could you please use here absolute path with the image registry that it should use? Usually there are some registried preconfigured on the system and it might not work in some scenarios.

Please use something like this: `docker.io/library/python:3`